.coderabbit.yaml

```diff
@@ -0,0 +1,77 @@
+language: en-US
+early_access: false
+reviews:
+  profile: chill
+  request_changes_workflow: false
+  high_level_summary: true
+  poem: false
+  review_status: true
+  collapse_walkthrough: false
+  auto_review:
+    enabled: true
+    drafts: true
+  pre_merge_checks:
+    custom_checks:
+      - name: "Go Error Handling"
+        mode: warning
+        instructions: |
+          Ensure proper Go error handling patterns:
+          - Never ignore errors with `_` without justification
+          - Errors should be wrapped with context using fmt.Errorf with %w
+          - Avoid panic() except in init() or fatal conditions
+          - Check for nil before dereferencing pointers
+
+      - name: "SQL Injection Prevention"
+        mode: error
+        instructions: |
+          Prevent SQL injection vulnerabilities:
+          - NEVER concatenate user input into SQL queries
+          - Always use parameterized queries with $1, $2, etc.
+          - Flag any string concatenation in SQL query construction
```

Comment:
we do that all the time, heh. wouldn't want that to be an error, but perhaps a warning?